lib/airbrussh/console_formatter.rb

```diff
@@ -69,7 +73,12 @@ def write(obj)
     def write_log_message(log_message)
       return if debug?(log_message)
       print_task_if_changed
-      print_indented_line(log_message.to_s)
+      verbosity = log_message.verbosity
+      # only write colored label for warning and above
+      return print_indented_line(log_message.to_s) if verbosity < 2
+      # use level string to call corresponding colorize method
+      level = send(LEVEL_COLORS[verbosity], LEVEL_NAMES[verbosity].ljust(6))
+      print_indented_line(level + log_message.to_s)
```

Comment:
The behavior looks good, but I think this code is a bit fragile because it depends on the verbosity levels being certain integer values. For example `if verbosity < 2` is not very clear when it could be written as `if verbosity < SSHKit::Logger::WARN`.

What do you think about using a `case` statement instead? Like this:

```ruby
def write_log_message(log_message)
  return if debug?(log_message)
  print_task_if_changed
  print_indented_line(format_log_message(log_message))
end

def format_log_message(log_message)
  case log_message.verbosity
  when SSHKit::Logger::WARN
    "WARN  #{yellow(log_message.to_s)}"
  when SSHKit::Logger::ERROR
    "ERROR #{red(log_message.to_s)}"
  when SSHKit::Logger::FATAL
    "FATAL #{red(log_message.to_s)}"
  else
    log_message.to_s
  end
end
```